tests/integration/blocks/test_flow.py

```diff
@@ -120,9 +113,7 @@ def run_block():
                 block_to_run.run_right()
 
             msgQueue.run_lambda(run_block)
-            time.sleep(0.1)
-            while block_to_run.is_running:
-                time.sleep(0.1)
+            time.sleep(2)
```

Comment:
Same as above. Probably something to do with `block_to_run.is_running` not updating properly. This is a bug.